### PR TITLE
Resolves #1710, #1709, #1708, #1707, #1706, #1705, #1704, #1703, #1702, #1701, #1700

### LINKED ISSUE #1710: The documentation for the -Replace parameter needs clarification.
You can modify more than one property by specifying a semicolon-separated list. The format should be:
-Replace @{Attribute1LDAPDisplayName=value[]; Attribute2LDAPDisplayName=value[]}

---
#### Document Details

⚠ *Do not edit this section. It is required for docs.microsoft.com ➟ GitHub issue linking.*

* ID: 850238a0-9221-119b-7088-bc3728b794aa
* Version Independent ID: 2ad8e7de-8de4-f913-3ba4-7dd806cfbc63
* Content: [Set-ADDomain (addsadministration)](https://docs.microsoft.com/en-us/powershell/module/addsadministration/set-addomain?view=win10-ps)
* Content Source: [docset/windows/addsadministration/Set-ADDomain.md](https://github.com/MicrosoftDocs/windows-powershell-docs/blob/live/docset/windows/addsadministration/Set-ADDomain.md)
* Product: **w10**
* Technology: **powershell-windows**
* GitHub Login: @andreabarr
* Microsoft Alias: **v-anbarr**

### PR DESCRIPTION
**Pull request by velkovb**
Fix -Replace parameter description.
https://github.com/MicrosoftDocs/windows-powershell-docs/issues/1710
https://github.com/MicrosoftDocs/windows-powershell-docs/issues/1709
https://github.com/MicrosoftDocs/windows-powershell-docs/issues/1708
https://github.com/MicrosoftDocs/windows-powershell-docs/issues/1707
https://github.com/MicrosoftDocs/windows-powershell-docs/issues/1706
https://github.com/MicrosoftDocs/windows-powershell-docs/issues/1705
https://github.com/MicrosoftDocs/windows-powershell-docs/issues/1704
https://github.com/MicrosoftDocs/windows-powershell-docs/issues/1703
https://github.com/MicrosoftDocs/windows-powershell-docs/issues/1702
https://github.com/MicrosoftDocs/windows-powershell-docs/issues/1701
https://github.com/MicrosoftDocs/windows-powershell-docs/issues/1700

### CODE REVIEW

**Comment by e0i** [comment]
@JohanFreelancer9 Copy editing is needed for this PR. Thanks.

**Comment by e0i** [comment]
@JohanFreelancer9 Minor style checks needed for this PR. Thanks.

**Comment by e0i** [comment]
@dariomws306 @get-itips @tseward

Peer review needed for this PR. Thanks.

File: docset/winserver2012-ps/activedirectory/Set-ADComputer.md [hunk 1]
```diff
@@ -39,92 +39,53 @@ Set-ADComputer [-WhatIf] [-Confirm] [-AuthType <ADAuthType>] [-Credential <PSCre
...
 You can modify commonly used property values by using the cmdlet parameters.
-Property values that are not associated with cmdlet parameters can be modified by using the Add, Replace, Clear and Remove parameters.
+Property values that are not associated with cmdlet parameters can be modified by using the *Add*, *Replace*, *Clear*, and *Remove* parameters.
 
-The Identity parameter specifies the Active Directory computer to modify.
-You can identify a computer by its distinguished name Members (DN), GUID, security identifier (SID) or Security Accounts Manager (SAM) account name.
-You can also set the Identity parameter to an object variable such as $\<localComputerObject\>, or you can pass an object through the pipeline to the Identity parameter.
-For example, you can use the Get-ADComputer cmdlet to retrieve a computer object and then pass the object through the pipeline to Set-ADComputer.
+The *Identity* parameter specifies the Active Directory computer to modify.
+You can identify a computer by its distinguished name, GUID, security identifier (SID) or Security Accounts Manager (SAM) account name.
```
Comment (JohanFreelancer9) [inline docset/winserver2012-ps/activedirectory/Set-ADComputer.md]:
```suggestion
You can identify a computer by its distinguished name, GUID, security identifier (SID), or Security Accounts Manager (SAM) account name.
```

File: docset/winserver2012r2-ps/addsadministration/Set-ADComputer.md [hunk 2]
```diff
@@ -50,45 +50,45 @@ Set-ADComputer [-WhatIf] [-Confirm] [-AuthType <ADAuthType>] [-Credential <PSCre
 ## DESCRIPTION
 The **Set-ADComputer** cmdlet modifies the properties of an Active Directory computer object.
 You can modify commonly used property values by using the cmdlet parameters.
-Property values that are not associated with cmdlet parameters can be modified by using the **Add**, **Replace**, **Clear**, and **Remove** parameters.
+Property values that are not associated with cmdlet parameters can be modified by using the *Add*, *Replace*, *Clear*, and *Remove* parameters.
 
-The **Identity** parameter specifies the Active Directory computer to modify.
+The *Identity* parameter specifies the Active Directory computer to modify.
 You can identify a computer by its distinguished name, GUID, security identifier (SID) or Security Accounts Manager (SAM) account name.
```
Comment (JohanFreelancer9) [inline docset/winserver2012r2-ps/addsadministration/Set-ADComputer.md]:
```suggestion
You can identify a computer by its distinguished name, GUID, security identifier (SID), or Security Accounts Manager (SAM) account name.
```

File: docset/winserver2012r2-ps/addsadministration/Set-ADFineGrainedPasswordPolicy.md [hunk 3]
```diff
@@ -44,18 +44,18 @@ Set-ADFineGrainedPasswordPolicy [-WhatIf] [-Confirm] [-AuthType <ADAuthType>] [-
 ## DESCRIPTION
 The **Set-ADFineGrainedPasswordPolicy** cmdlet modifies the properties of an Active Directory fine-grained password policy.
 You can modify commonly used property values by using the cmdlet parameters.
-Property values that are not associated with cmdlet parameters can be modified by using the **Add**, **Replace**, **Clear**, and **Remove** parameters.
+Property values that are not associated with cmdlet parameters can be modified by using the *Add*, *Replace*, *Clear*, and *Remove* parameters.
 
-The **Identity** parameter specifies the Active Directory fine-grained password policy to modify.
+The *Identity* parameter specifies the Active Directory fine-grained password policy to modify.
 You can identify a fine-grained password policy by its distinguished name, GUID or name.
```
Comment (JohanFreelancer9) [inline docset/winserver2012r2-ps/addsadministration/Set-ADFineGrainedPasswordPolicy.md]:
```suggestion
You can identify a fine-grained password policy by its distinguished name, GUID, or name.
```

File: docset/winserver2012r2-ps/addsadministration/Set-ADServiceAccount.md [hunk 4]
```diff
@@ -47,18 +47,18 @@ Set-ADServiceAccount [-WhatIf] [-Confirm] [-AuthType <ADAuthType>] [-Credential
...
+Property values that are not associated with cmdlet parameters can be modified by using the *Add*, *Remove*, *Replace*, and *Clear* parameters.
 
-The **Identity** parameter specifies the Active Directory MSA to modify.
+The *Identity* parameter specifies the Active Directory MSA to modify.
 You can identify an MSA by its distinguished name, GUID, security identifier (SID), or Security Account Manager (SAM) account name.
-You can also set the **Identity** parameter to an object variable such as **$\<localServiceAccountObject\>**, or you can pass an object through the pipeline to the **Identity** parameter.
-For example, you can use the Get-ADServiceAccount cmdlet to retrieve a MSA object and then pass the object through the pipeline to the Set-ADServiceAccount cmdlet.
+You can also set the *Identity* parameter to an object variable such as `$<localServiceAccountObject>`, or you can pass an object through the pipeline to the *Identity* parameter.
+For example, you can use the **Get-ADServiceAccount** cmdlet to retrieve a MSA object and then pass the object through the pipeline to the **Set-ADServiceAccount** cmdlet.
```
Comment (JohanFreelancer9) [inline docset/winserver2012r2-ps/addsadministration/Set-ADServiceAccount.md]:
```suggestion
For example, you can use the **Get-ADServiceAccount** cmdlet to retrieve an MSA object and then pass the object through the pipeline to the **Set-ADServiceAccount** cmdlet.
```

**Comment by velkovb** [comment]
> Only one comment. Why you use User0x as domain example instead of Fabrikam? I think User0x is confusing and Fabrikam or Contoso is better because they are well know domain examples.

@dariomws306 I was just reusing the example that was already there. I could change it to the well known ones :)

**Comment by dariomws306** [comment]
> > Only one comment. Why you use User0x as domain example instead of Fabrikam? I think User0x is confusing and Fabrikam or Contoso is better because they are well know domain examples.
> 
> @dariomws306 I was just reusing the example that was already there. I could change it to the well known ones :)

I saw at least one example that you have replaced Fabrikam with User0x. But I can be wrong. Maybe it's not critical but I think it's a good idea. Let's double check with @get-itips and @tseward.
Also, I think it's not a requirement to change all the examples now, just the examples that you apply some updates.

**Comment by e0i** [comment]
@tseward Ready to proceed with your go.

**Comment by velkovb** [comment]
True, I get the examples from the 2016 version and make them the same in 2012R2 and 2012. So it is possible I have replaced it but I think examples should be consistent across versions.

**Pull request merged**